<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color:rgb(57, 57, 57);</string>
      </property>
      <widget class="QLabel" name="label1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white
</string>
       </property>
       <property name="text">
        <string>Enter the Function</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="eqInput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>331</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter Function</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>331</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px;
    border-color:white;
    border-style: solid;</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>Set Values</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="minX">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>137</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>min X</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="maxX">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>137</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;

background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="placeholderText">
         <string>max X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>Max X Value</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>Min X Value</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px;
    border-color:white;
    border-style: solid;</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>Set Labels</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>Y Label</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border-color: rgb(57, 57, 57);
</string>
        </property>
        <property name="text">
         <string>X Label</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="xlabel">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>211</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ylabel">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="plot">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>610</y>
         <width>191</width>
         <height>45</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 255);
border-radius:10px;
color:white;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
align:center
</string>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="MplWidget" name="MplWidget" native="true">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>120</y>
         <width>631</width>
         <height>611</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
margin-top : 10px
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>670</y>
         <width>191</width>
         <height>50</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 0);
border-radius:10px;

color:white</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Function Plotter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>widget_4</zorder>
      <zorder>widget_3</zorder>
      <zorder>label1</zorder>
      <zorder>eqInput</zorder>
      <zorder>plot</zorder>
      <zorder>MplWidget</zorder>
      <zorder>clear</zorder>
      <zorder>label_7</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
